src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -96,6 +96,17 @@ public async Task<ActionResult<IEnumerable<DataFileInfo>>> GetDataFileInfo(Guid
                 .HandleFailuresOrOk();
         }
 
+        [HttpPut("release/{releaseId}/data/order")]
+        [Produces("application/json")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(404)]
+        public async Task<ActionResult<IEnumerable<DataFileInfo>>> ReorderDataFiles(Guid releaseId,
+            Dictionary<Guid,int> newOrder)
```

Comment:
I did it this way because I was copying how it was done in `ContentService#ReorderContentSections`.

The change was easy to make so I've switched to using a list of fileIds now instead of a dictionary.